### ISSUE TITLE
[Request] Selective Sound Input

### ISSUE THREAD

**Issue by Deadzulu** [FlavioFS/ParsecSoda#41]
Essentially just choose what apps can and can't play sound into parsec soda.

**Comment by Engoni** [comment]
Yeah, the regular parsec app has a new future like that and can exclude discord, hope it come to soda too!
